src/main/java/com/softserveinc/ita/pageobjects/GoogleSearchPage.java

```diff
@@ -0,0 +1,37 @@
+package com.softserveinc.ita.pageobjects;
+
+import Elements.Img;
+import Elements.Input;
+import Elements.Link;
+import org.openqa.selenium.By;
+
+import java.util.List;
+
+public class GoogleSearchPage extends AbstractBasePage {
+
+    private final Input googleInput = new Input(By.name("q"));
+    private final Input searchButton = new Input(By.name("btnK"));
+    private final Link searchResults = new Link(By.xpath("//div[@class='yuRUbf']/a"));
+    private final Link followLink = new Link(By.xpath("//a[contains(@href, 'rozetka.com')]"));
+
+    public GoogleSearchPage() {
```

Comment:
according to the changes I have proposed, there will be such changes, also in the elements

```
    public GoogleSearchPage(final WebDriver driver) {
        super(driver);
    }

    @Override
    public boolean isPageOpened() {
        return new Img(driver, By.xpath("//img[@alt='Google']")).isElementsDisplayed();
    }
```